packages/odyssey-react/src/components/Checkbox/Checkbox.theme.ts

```diff
@@ -0,0 +1,44 @@
+/*!
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import type { ThemeReducer } from "@okta/odyssey-react-theme";
+
+export const theme: ThemeReducer = (theme) => ({
+  ColorBorder: theme.ColorBorderUi,
+  ColorBorderChecked: theme.ColorPrimaryBase,
+  ColorBorderHover: theme.ColorPrimaryBase,
+  ColorBorderFocus: theme.ColorPrimaryBase,
+  ColorBorderDisabled: theme.ColorBorderDisabled,
+  ColorBorderDisabledChecked: theme.ColorPrimaryLight,
+  ColorBorderInvalid: theme.ColorBorderDanger,
+  ColorBorderInvalidHover: theme.ColorDangerDark,
+  ColorBackground: theme.ColorPrimaryBase,
+  ColorBackgroundDisabled: theme.ColorBackgroundDisabled,
+  ColorBackgroundDisabledChecked: theme.ColorPrimaryLight,
+  ColorBackgroundInvalid: theme.ColorBackgroundDangerDark,
+  ColorCheck: theme.ColorTextBodyInverse,
+  ColorTextDisabled: theme.ColorTextBody,
+  ColorTextInvalid: theme.ColorTextDanger,
+  ColorFocusOutline: theme.ColorFocusPrimary,
+  ColorFocusOutlineInvalid: theme.ColorFocusDanger,
+  FocusOutlineWidth: theme.FocusOutlineWidth,
+  MarginBottom: theme.SpaceRemXs,
```

Comment:
Would you mind making some of these local variables a bit more specific to use?

When viewing the module, I prefer `margin-bottom: var(--SpaceRemXs)` because it answers "what value is being applied here?"

However, viewing the theme, I see the value of `PaddingLabelLeft`. It answers "what does this variable affect?"

`MarginBottom`, on the other hand, feels non-specific to the place it's utilized.